ice_setup/ice.py

```diff
@@ -1074,17 +1098,21 @@ def parse_args(self):
         sudo_check()
 
         if parser.has('all'):
-            repo_path = parser.get('all')
-            configure_local(repo_path=repo_path)
-            configure_remotes(repo_path=repo_path)
+            package_path = parser.get('all')
+            configure_local('calamari-server', package_path or CWD)
```

Comment:
I think I did one better -- rather than putting the `path or CWD` logic in the configure_* functions, I was able to put it one common place -- get_package_source().  Now that CWD isn't used so much, I could revert from CWD being a global var if you like.  I just didn't want to call os.getcwd() all over the place.  But now it would only be in two places, I think.

Can I get a re-review on this?